packages/mocks/http.js

```diff
@@ -6,22 +6,68 @@ const VERSION = Symbol.for('msw-integration-layer::version');
  * and it's import should be pure and non-sideeffectful.
  */
 export const http = {
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   get: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'get', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   post: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'post', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   put: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'put', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   patch: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'patch', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   delete: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'delete', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
   options: (endpoint, handler) => {
     return { [VERSION]: '1.x.x', method: 'options', endpoint, handler };
   },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
+  all: (endpoint, handler) => {
+    return { [VERSION]: '1.x.x', method: 'all', endpoint, handler };
+  },
+  /**
+   * @param {string} endpoint
+   * @param {import('./types.js').handler} handler
+   * @returns {import('./types.js').Mock}
+   */
+  head: (endpoint, handler) => {
```

Comment:
I dont think MSW supports `head`, so this will cause an error

Reply:
it's in msw's rest
https://github.com/mswjs/msw/blob/v0.0.0-fetch.rc-15/src/core/rest.ts#L35

Reply:
Interesting, its not documented on this page: https://mswjs.io/docs/api/rest

Ok, lets keep it 👍

Reply:
Make sure to add `'head'` here: https://github.com/modernweb-dev/web/pull/2353/files#diff-dfdc098b54356afdd499c56eec03801b27a275a1436dbe8cf6d77eb468ae70a3R3

Reply:
good catch - added 👍